app/views/root/_gem_base.html.erb

```diff
@@ -46,7 +46,6 @@
   global_bar: global_bar,
   homepage: homepage_blue_navbar,
   logo_link: logo_link,
-  product_name: product_name,
```

Comment:
This line is duplicated below, so I'm removing it. It caused a warning from Ruby.